packages/Radio/stories/examples/Controlled.js

```diff
@@ -1,49 +1,32 @@
 import React from "react";
 import { action } from "@storybook/addon-actions";
-import { Rule } from "storybook/assets/styles/common.styles";
-import Heading from "@paprika/heading";
-import { RadioStory } from "../Radio.stories.styles";
+import { Gap } from "storybook/assets/styles/common.styles";
+import StoryHeading from "storybook/components/StoryHeading";
 import Radio from "../../src/Radio";
 
-export const radioItems = ["Radio 1", "Radio 2 label", "Radio 3 option"];
+const radioItems = ["Radio 1", "Radio 2", "Radio 3"];
 
-const ExampleStory = props => {
+const ExampleStory = () => {
   const [checkedIndex, setCheckedIndex] = React.useState(1);
-  const handleIndexChange = e => {
-    setCheckedIndex(Number(e.target.value));
-  };
+
+  function handleIndexChange(event) {
+    setCheckedIndex(Number(event.target.value));
+  }
+
+  function handleRadioChange(activeIndex) {
+    setCheckedIndex(activeIndex);
+    action("Radio selection changed to index ")(activeIndex);
```

Comment:
is the space on purpose?

Reply:
Yeah, it will display the index after that string, so it is easier to read this way